modules/sensor-upgrade-section.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="sensor-upgrade-section_{context}"]
+= Sensor upgrade section
+
+The *Sensor upgrade* section organizes the information into the following groups:
+
+* *Up to date*: Indicates that Sensor is running the latest supported version.
```

Comment:
Per style guide (if the GUI has no hyphens, defer to the GUI use but consider mentioning this to engineering)
```suggestion
* *Up-to-date*: Indicates that Sensor is running the latest supported version.
```